xmodule/library_content_block.py

```diff
@@ -436,15 +504,11 @@ def author_view(self, context):
                 max_count = self.max_count
                 if max_count < 0:
                     max_count = len(self.children)
-
-                fragment.add_content(self.runtime.service(self, 'mako').render_cms_template(
-                    "library-block-author-preview-header.html", {
-                        'max_count': max_count,
-                        'display_name': self.display_name or self.url_name,
-                    }))
```

Comment:
Removing this fragment would remove the message:

> Showing all matching content eligible to be added into My Static Library Content. Each student will be assigned $max_count components drawn randomly from this list.

That seems like a helpful message, why not keep it, but with extra parameter that lets you adjust it when `manual==True`?